app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -134,14 +134,15 @@ sealed class DaxDialogCta(
             )
 
         override fun getDaxText(context: Context): String {
-            val trackersFiltered = trackers.asSequence()
-                .filter { it.entity?.isMajor == true }
-                .map { it.entity?.displayName }
+            val trackers = trackers.asSequence()
+                .mapNotNull { it.entity }
+                .sortedByDescending { it.prevalence }
```

Comment:
Is this the same logic that should be applied to the logos? If we apply any change here, should it be applied to the other logic too? if so, maybe we should consider some common logic, or one to rely on a list already sorted by the a domain component. (e.g: if we already sorted the tracker events, do we need to sort it again when rendering?)

Reply:
This logic now happens in the `Site` as an extension function that is then used to pass a list of entities rather than a list of tracking events.